samples/MvcSandbox/package.json

```diff
@@ -1,4 +1,5 @@
 {
   "name": "ASP.NET",
+  "private":  true,
```

Comment:
@ryanbrandenburg I didn't need to force this change into a commit. Why isn't the `.gitignore` file you added working as expected?